ODIN_II/regression_test/benchmark/task/operators/simulation_result.csv

```diff
@@ -183,11 +183,7 @@ operators/twobits_logical_less_than/k6_frac_N10_frac_chain_mem32K_40nm         ,
 operators/twobits_logical_less_than/no_arch                                    , n/a                                   , twobits_logical_less_than.blif         , 0   , 10.6        , 6.2          , 2.2                , 100.0           , 1            , 5 , 2  , 11           , 1    , -1   , -1        , -1    , -1     , -1                , 9           , 4           , 11            , 13        
-operators/unary_bitwise_bufnode/k6_frac_N10_frac_chain_mem32K_40nm             , k6_frac_N10_frac_chain_mem32K_40nm.xml, unary_bitwise_bufnode.blif             , 0   , 23.7        , 58.8         , 0.6                , 100.0           , 1            , 2 , 2  , 6            , 1    , -1   , -1        , -1    , -1     , 4                 , 8           , 4           , 6             , 8         
-operators/unary_bitwise_bufnode/k6_N10_40nm                                    , k6_N10_40nm.xml                       , unary_bitwise_bufnode.blif             , 0   , 13.4        , 9.1          , 0.6                , 100.0           , 1            , 2 , 2  , 6            , 1    , -1   , -1        , -1    , -1     , 6                 , 8           , 4           , 6             , 8         
-operators/unary_bitwise_bufnode/k6_N10_mem32K_40nm                             , k6_N10_mem32K_40nm.xml                , unary_bitwise_bufnode.blif             , 0   , 23.2        , 53.1         , 0.6                , 100.0           , 1            , 2 , 2  , 6            , 1    , -1   , -1        , -1    , -1     , 6                 , 8           , 4           , 6             , 8         
-operators/unary_bitwise_bufnode/no_arch                                        , n/a                                   , unary_bitwise_bufnode.blif             , 0   , 10.7        , 3.7          , 0.6                , 100.0           , 1            , 2 , 2  , 6            , 1    , -1   , -1        , -1    , -1     , -1                , 8           , 4           , 6             , 8         
-operators/unary_bitwise_not/k6_frac_N10_frac_chain_mem32K_40nm                 , k6_frac_N10_frac_chain_mem32K_40nm.xml, unary_bitwise_not.blif                 , 0   , 24.0        , 55.9         , 0.4                , 100.0           , 1            , 2 , 2  , 7            , 1    , -1   , -1        , -1    , -1     , 4                 , 8           , 4           , 7             , 9         
+operators/unary_bitwise_not/k6_frac_N10_frac_chain_mem32K_40nm                 , k6_frac_N10_frac_chain_mem32K_40nm.xml, unary_bitwise_not.blif                 , 0   , 24.0        , 55.9         , 0.4                , 100.0           , 1            , 2 , 2  , 7            , 1    , -1   , -1        , -1    , -1     , 4                 , 8           , 4           , 7             , 9
```

Comment:
@jeanlego How do I regenerate these results?

Reply:
I use the Makefile `qor` target but you can run it by hand like so
https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/89ffdc1ff8e7a27e6321926826badd78b8a30ec0/ODIN_II/Makefile#L115-L127